docs/blog/2019-01-01-publish-multiple-gatsby-sites/index.md

```diff
@@ -461,9 +461,9 @@ Hopefully what I’ve said makes more sense with the code in front of us. The `s
 
 ### Now
 
-Now is a tool created by ZEIT that makes the process of deploying Node applications simple. Now has recently released updates to the way they deploy static websites. Fortunately our Gatsby packages build to static sites, so it’s a win for us.
+Now is a tool created by Vercel(formally ZEIT) that makes the process of deploying Node applications simple. Now has recently released updates to the way they deploy static websites. Fortunately our Gatsby packages build to static sites, so it’s a win for us.
 
-If you haven’t used Now before, then go-ahead and [get started](https://zeit.co/now#features). This will install the `now-cli` and get a free account created, things we need for the tutorial. The getting started page also has a brief FAQs section that’s well worth reading.
+If you haven’t used Now before, then go-ahead and [get started](https://vercel.com/home#features). This will install the `now-cli` and get a free account created, things we need for the tutorial. The getting started page also has a brief FAQs section that’s well worth reading.
```

Comment:
```suggestion
If you haven’t used Vercel before, then go-ahead and [get started](https://vercel.com/home#features). This will install the `now-cli` and get a free account created, things we need for the tutorial. The getting started page also has a brief FAQs section that’s well worth reading.
```

and much more on this page :)